test/per_file_coverage.sh

```diff
@@ -53,6 +53,7 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/extensions/filters/http/lua:96.4"
 "source/extensions/filters/http/oauth2:96.0"
 "source/extensions/filters/http/wasm:1.9"
+"source/extensions/filters/http/rate_limit_quota:70" # TODO(tyxia) WIP. Improve coverage as development continues
```

Comment:
I believe this was part of brekaing coverage for main
I'm inclined to roll this and https://github.com/envoyproxy/envoy/pull/24304 back as this is new code.
@yanavlasov WDYT?

Reply:
I noticed the main coverage breakage before as well in my previous commits in this PR(from 96%.1 to 96%). Then I fixed it by increasing the coverage of this extension to  [83.1%](https://storage.googleapis.com/envoy-pr/0faa669/coverage/index.html) even though it is still listed as 70 in this check file.

So I think it should be fine if we are referring to same main coverage break. Please let me know if you have any questions @alyssawilk

Reply:
I have sent out a PR for your review and left the comment on open issue.

I will resolved the conversation here. And please let me know if you have any questions or requests on that PR or open issue. Thanks!